data/src/main/java/me/androidbox/data/local/entity/EventEntity.kt

```diff
@@ -16,7 +16,7 @@ data class EventEntity(
     val remindAt: Long,
     val eventCreatorId: String,
     val isUserEventCreator: Boolean,
-    val isGoing: Boolean,
+    val host: String,
```

Comment:
I removed the isGoing as I noticed in the /agenda, create /event, and get /event there is no isGoing. But here is a `host`